docusaurus/docs/React/hooks/message-hooks.mdx

```diff
@@ -179,6 +179,26 @@ const MyCustomMessageComponent = () => {
 };
 ```
 
+### useReactionsFetcher
+
+A [custom hook](https://github.com/GetStream/stream-chat-react/blob/master/src/components/Message/hooks/useReactionsFetcher.ts) to handle loading message reactions. Returns an async function that loads and returns message reactions.
+
+```jsx
+const MyCustomMessageComponent = () => {
```

Comment:
Maybe I would be showing how to override `ReactionsList` rather than a whole message.

Reply:
I don't think that's possible - AFAIK you cannot override a component used for `ReactionsList`, you have to provide a whole custom message component instead.

Reply:
@MartinCupela I stand corrected, will update the doc.